BSP/Inc/BSP_SPI.h

```diff
@@ -16,6 +17,23 @@
  * @return  None
  */
 void BSP_SPI_Init(void);
+#endif
+
+#ifdef RTOS
+/**
+ * @note    If other ICs need to be connected onto the SPI line, this code will have to be modified.
+ */
+
+/**
+ * @brief   Initializes the SPI port connected to the LTC6820.
+ *          This port communicates with the LTC6811 voltage and temperature
+ *          monitoring IC. The LTC6820 converts the SPI pins to 2-wire isolated SPI.
+ *          Look at analog devices website and LTC6811's or LTC6820's datasheets.
+ * @param   None
+ * @return  None
+ */
+void BSP_SPI_Init(bsp_os_t *spi_os);
```

Comment:
Instead of having this ifdef, just make it so that way the function always accepts an input of that type. When we run the baremetal version of the code, we can just load the pend and post functions with empty code. It's neater that way.